components/NodeBox/NodeBox.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react'
 import PointerBox from '../PointerBox/PointerBox'
 import ValueBox from '../ValueBox/ValueBox'
 import css from 'styled-jsx/css'
-// TODO: Make files structure consistent (here ..Arrow/index.tsx)
 import Arrow from '../Arrow/Arrow'
+// TODO: Make files structure consistent (here ..Arrow/index.tsx)
 
 type Props = {
   pointer?: boolean
   pointed?: boolean
   orientation?: 'row' | 'column'
+  arrowOrientation?: 'string'
   color?: number[]
```

Comment:
I think we could use some library here that will take care of those shades out of the box, instead of passing those numbers, for now it can stay